03-Node-npm-modules/movie.js

```diff
@@ -0,0 +1,31 @@
+var Director = require('./director.js');
+
+var Movie = function () {
+  this.setName(name);
+  this.director = new Director();
+};
+
+Movie.prototype = {
+	constructor: Movie,
+
+	setName = function (attr, value) {
+  		this.attributes = attr || 'Unknown';
```

Comment:
attributes will always has name, right? It would be better to define a more explicit name property then.